public/app/percona/shared/core/reducers/index.ts

```diff
@@ -384,5 +387,6 @@ export default {
     templates: templatesReducer,
     services: servicesReducer,
     backupLocations: perconaBackupLocations,
+    telemetry: uiEventsReducer
```

Comment:
If telemetry is disabled, events will be collected, and they will not be cleaned up because cleanup is happening on send.